src/styles/Blocks/nav.scss

```
@@ -0,0 +1,53 @@
+$main-color: #00acdc;
+$main-background-color: #fff;
+
+.nav__list {
+  display: flex;
+}
+
+.nav__item {
+  display: flex;
+  height: 60px;
+  margin-right: 20px;
+  list-style: none;
+  text-transform: uppercase;
+  font-family: Roboto, sans-serif;
+  font-weight: 500;
+  font-size: 12px;
+
+  &:last-child {
+    margin-right: 0;
+  }
+}
+
+.nav__link {
+  display: flex;
+  align-items: center;
+  text-decoration: none;
+  color: #060b35;
+  position: relative;
```

it is better to declare the positioning properties first
```suggestion
.nav__link {
  position: relative;
  display: flex;
  align-items: center;
  text-decoration: none;
  color: #060b35;
```

---

+ fixed it